### ISSUE TITLE
One publication can't save

### ISSUE THREAD

**Issue by Elisagig** [Mergpijp/GlobalOrthodoxy#219]
Dear Wessel,

This publication can't save at all. We want to replace "School Book" with "Youth-N" under genres but it is not working. It opens but can't save
https://globalorthodoxy.cls.ru.nl/publication/622/edit/?next=/?order_by=publication_city__name&direction=desc

**Comment by Woseseltops** [comment]
I did some digging in the code, and it turns out the 'contact website' field is broken. I've removed the value there (it was `www.zangak.am`), and now it works again. So the real task for this issue, I think, is to repair the 'contact website' field.

**Comment by Woseseltops** [comment]
Did some more digging, and my theory right now is that it is not broken, but that it's doing very strict validation: it really requires a full URL of the format `http://www.zangak.am`, just `www.zangak.am` is not good enough. There are now 2 approaches we can take:

* Teach the users to always enter full URLS. The web application is already helping out a bit with this, because it keeps showing 
![image](https://user-images.githubusercontent.com/1909317/156740437-236175ff-caf2-437a-bf41-9cbb91c169a4.png) until the format is correct
* Remove the validation, and just make this a free text field

Let me know what you prefer @Elisagig

**Comment by Elisagig** [comment]
Hi Wessel,
Sorry, I do not understand the part about removing a validation and making a free text field.

**Comment by Woseseltops** [comment]
Let me rephrase :).

Current situation: if you put something in the 'contact website' field, it MUST start with `http://` and contain a dot.
Suggested approach 2: everything you put in the 'contact website' field is accepted.

Does that make things clearer?
